independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/MultiInvoker.java

```diff
@@ -34,34 +36,86 @@ public MultiInvoker(WebTargetImpl target) {
         return (Multi<R>) super.get(responseType);
     }
 
+    /**
+     * We need this class to work around a bug in Mutiny where we can register our cancel listener
+     * after the subscription is cancelled and we never get notified
+     */
+    static class MultiRequest<R> {
```

Comment:
So can we drop this when the said mutiny bug is fixed?

Reply:
Yes, I'll add a note as to which bug it is: https://github.com/smallrye/smallrye-mutiny/issues/417

Reply:
Cool, thanks

Reply:
Added note.